src/Frontend/Modules/Profiles/Actions/Login.php

```diff
@@ -58,6 +58,17 @@ private function validateForm(): bool
 
         $loginStatus = FrontendProfilesAuthentication::getLoginStatus($txtEmail->getValue(), $txtPassword->getValue());
 
+        $profileId = FrontendProfilesModel::getIdByEmail($txtEmail->getValue());
+        if ($loginStatus === FrontendProfilesAuthentication::LOGIN_INVALID && $profileId !== 0) {
+            $loginAttempts = (int) FrontendProfilesModel::getSetting($profileId, 'login_attempts');
+
+            FrontendProfilesModel::setSetting($profileId, 'login_attempts', ++$loginAttempts);
+            if ($loginAttempts >= 10) {
+                FrontendProfilesModel::update($profileId, ['status' => FrontendProfilesAuthentication::LOGIN_BLOCKED]);
+                $loginStatus = FrontendProfilesAuthentication::LOGIN_BLOCKED;
```

Comment:
Move this variable declaration a line above and you can reduce the code to
`FrontendProfilesModel::update($profileId, ['status' => $loginStatus]);`

Reply:
Fixed 28a49acc4